issue-tracker/src/components/main/main.component.jsx

```diff
@@ -1,36 +1,71 @@
 import React, { Component } from "react";
-import Tabs from '../tabs/tabs.component';
-import Issues from '../issues/issues.component';
+import Tabs from "../tabs/tabs.component";
+import Issues from "../issues/issues.component";
 
-import './main.styles.css';
+import "./main.styles.css";
 
-import PROJECT_DATA from '../../data/project.data';
+import { PROJECT_DATA, client_uuid, herokuApi } from "../../data/project.data";
 
 class Main extends Component {
-    constructor(props) {
-      super(props);
-    
-      this.state = {
-        data: PROJECT_DATA
-      }
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      data: null
+    };
+  }
+
+  fetchProjects() {
+    if (
+      localStorage.getItem(client_uuid) &&
+      localStorage.getItem(client_uuid) !== ""
```

Comment:
If I remember correctly `if(localStorage.getItem(clien_uuid)` should be false if there is nothing there, much like when a parameter is null in normal JS